packages/edit-site/src/components/add-new-template/new-template.js

```diff
@@ -41,6 +73,8 @@ export default function NewTemplate( { postType } ) {
 			defaultTemplateTypes: select(
 				editorStore
 			).__experimentalGetDefaultTemplateTypes(),
+			defaultBlockTemplate: select( editorStore ).getEditorSettings()
```

Comment:
This was initially an API suited for "custom templates" and not "hierarchy templates". Is it fine to use it consistently no matter the template? 

Also should we sync the logic here with the fallback done in the template mode? (post editor)